### PR TITLE
Clean up `Interoperability` page.

### PR DESCRIPTION
**Pull request by Bosch-0**
Tidied up the file names changing from wallet-interoperability -> interoperability. No copy changes.

### CODE REVIEW

**Comment by Bosch-0** [comment]
Fixing some broken links now

**Comment by GBKS** [comment]
Sorry, but this is a mess to review now. Github interprets your changes as deletion of one page and addition of another. So I can't use the Github diff tools to see what has changed and need to pull up two browser windows next to each other and compare the text line by line. Could you please split this up into multiple PRs? I'd recommend one PR for the organizational stuff (renaming the page and moving files), and one for the actual copy tweaks.

**Comment by Bosch-0** [comment]
My bad, on to it!

**Comment by sbddesign** [comment]
I'll hold off on review until this is out of draft mode.

**Comment by Bosch-0** [comment]
@sbddesign @GBKS this PR focuses on file name changes only now, no new content is added.

File: guide/designing-products/interoperability.md [hunk 1]
```diff
@@ -4,32 +4,27 @@ title: Interoperability
...
-permalink: /guide/designing-products/wallet-interoperability/
+permalink: /guide/designing-products/interoperability/
+redirect_from: /guide/designing-products/wallet-interoperability/
 main_classes: -no-top-padding
-image: https://bitcoin.design/assets/images/guide/designing-products/wallet-interoperability/wallet-interoperability-preview.jpg
+image: https://bitcoin.design/assets/images/guide/designing-products/interoperability/wallet-interoperability-preview.jpg
 ---
 
 <!--
-
```
Comment (GBKS) [inline guide/designing-products/interoperability.md:13]: Why delete the white space? Much harder to read now.

Comment (Bosch-0) [reply guide/designing-products/interoperability.md:13]: That wasn't intentional, I overwrite the old MD file with the new and seems to have done that automatically, fixed.

**Review by GBKS — APPROVED** [review]
Looks good. Thank you for going through the effort and simplifying the PR. Super easy to review now.

**Pull request merged**